Dockerfile

```diff
@@ -36,4 +38,15 @@ WORKDIR /
 RUN npm run setup-chrome
 
 WORKDIR /app
-RUN zip ../interceptor-chrome.zip -r .
\ No newline at end of file
+RUN zip ../interceptor-chrome.zip -r .
+
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.name="The Interceptor Browser Extenstion" \
+      org.label-schema.description="TheInterceptor" \
+      org.label-schema.url="https://dark.florist/" \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.vcs-url="https://github.com/DarkFlorist/TheInterceptor.git" \
+      org.label-schema.vendor="Dark Florist" \
+      org.label-schema.version=$VERSION \
+      org.label-schema.schema-version="1.0"
```

Comment:
Similar to comment above, these seem unnecessary as the docker image is only used transiently during the build process but it is not published/retained.